src/main/java/net/coderbot/iris/pipeline/newshader/fallback/ShaderSynthesizer.java

```diff
@@ -88,19 +89,19 @@ public static String vsh(boolean hasChunkOffset, ShaderAttributeInputs inputs, F
 
 				shader.append("in vec3 Normal;\n");
 
-				main.append("    vertexColor = minecraft_mix_light(Light0_Direction, Light1_Direction, Normal, Color * ColorModulator);\n");
+				main.append("    iris_vertexColor = minecraft_mix_light(Light0_Direction, Light1_Direction, Normal, Color * ColorModulator);\n");
```

Comment:
This doesn't seem quite right... Are you sure that the mix_light function won't care about the alpha value here? If so, add a comment stating that.